br/pkg/lightning/restore/check_info.go

```diff
@@ -397,12 +397,12 @@ func (rc *Controller) estimateSourceData(ctx context.Context) (int64, error) {
 			continue
 		}
 		for _, tbl := range db.Tables {
+			originSource += tbl.TotalSize
 			tableInfo, ok := info.Tables[tbl.Name]
 			if ok {
-				originSource += tbl.TotalSize
 				// Do not sample small table because there may a large number of small table and it will take a long
 				// time to sample data for all of them.
-				if tbl.TotalSize < int64(config.SplitRegionSize) {
+				if rc.cfg.TikvImporter.Backend != config.BackendLocal || tbl.TotalSize < int64(config.SplitRegionSize) {
```

Comment:
i think the sampling is useful for importer backend  too

```suggestion
				if rc.cfg.TikvImporter.Backend == config.BackendTiDB || tbl.TotalSize < int64(config.SplitRegionSize) {
```